packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -2,52 +2,46 @@ import chalk from 'chalk';
 import fs from 'fs';
 import path from 'path';
 import { prompt } from 'inquirer';
-import { ParsedArgs } from 'minimist';
-import { NextjsAnswer } from './NextjsAnswer';
-import { userPrompts } from './user-prompts';
+import { prompts, NextjsAnswer, Prerender } from './prompts';
 import { Initializer } from '../../common/Initializer';
 import { transform } from '../../common/steps';
 import { isDevEnvironment } from '../../common/utils/helpers';
 import { removeDevDependencies } from './remove-dev-dependencies';
+import { NextjsArgs } from './args';
+import { FetchWith } from '../../common/prompts/base';
 
 export class NextjsInitializer implements Initializer {
-  async init(args: ParsedArgs) {
-    // identify defaults
-    let defaults = args.yes
+  async init(args: NextjsArgs) {
+    const defaults = args.yes
       ? {
           appName: 'sitecore-jss-nextjs',
-          destination: `${process.cwd()}\\sitecore-jss-nextjs`,
```

Comment:
Do we want to keep this default when using --yes to override the generic default location?

Reply:
We have a question at the start:
https://github.com/Sitecore/jss/blob/diy-answers-prompts/packages/create-sitecore-jss/src/index.ts#L45-L60
in this case, it should be handled here. And I think that this value equals the default value in the question, it doesn't make sense to duplicate it

Reply:
Yes, fair enough! Was just wondering if when using --yes the default destination would match the framework name, but it's not necessary.